tracer/src/Datadog.Trace/Ci/IntelligentTestRunnerClient.cs

```diff
@@ -797,6 +810,114 @@ async Task<EarlyFlakeDetectionResponse> InternalGetEarlyFlakeDetectionTestsAsync
         }
     }
 
+    public async Task<ImpactedTestsDetectionResponse> GetImpactedTestsDetectionFilesAsync()
+    {
+        Log.Debug("ITR: Getting impacted tests detection modified files...");
+        var framework = FrameworkDescription.Instance;
+        var repository = await _getRepositoryUrlTask.ConfigureAwait(false);
+        var branch = await _getBranchNameTask.ConfigureAwait(false);
+        var currentSha = await _getShaTask.ConfigureAwait(false);
```

Comment:
These also changed recently, now there's no task for those, it's already executed in the .ctor